worker/undertaker/manifold_test.go

```diff
@@ -106,6 +110,22 @@ func (s *manifoldSuite) TestNewFacadeError(c *gc.C) {
 	c.Check(worker, gc.IsNil)
 }
 
+func (s *manifoldSuite) TestNewCredentialAPIError(c *gc.C) {
```

Comment:
Should this test be on every manifold using a cred validator facade?
I guess we don't really do this depth of testing anywhere else.

Reply:
Not really as different workers create and use it slightly differently.